arlo_server/models.py

```diff
@@ -361,3 +361,64 @@ class ProcessingStatus(str, Enum):
     PROCESSING = "PROCESSING"
     PROCESSED = "PROCESSED"
     ERRORED = "ERRORED"
+
+
+class USState(str, Enum):
```

Comment:
This looks great.

Is there no way to actually use this in the db model? Or did you just choose to not validate at the db level?

Reply:
I chose not to validate at the db level.